### PR TITLE
md file

### LINKED ISSUE #3: Kelvinmpham adds file about Pasadena
## This is a tracking issue
- [ ] Task 1
- [ ] Task 2

### PR DESCRIPTION
**Pull request by kelvinmpham**
This pull request fixes #3

### CODE REVIEW

File: kelvinmpham.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Kelvinmpham
+
+##Header 1
+
+##Header 2
```
Comment (kelvinmpham) [inline kelvinmpham.md]: comment

File: kelvinmpham.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Kelvinmpham
```
Comment (kelvinmpham) [inline kelvinmpham.md]: comm

**Review by kelvinmpham — COMMENTED** [review]
👍

File: kelvinmpham.md (hunk 2 above)
Comment (i3ince) [reply kelvinmpham.md]: ent

Comment (i3ince) [reply kelvinmpham.md]: needs more emoji

File: kelvinmpham.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+##Header 1
+
```
Comment (nttoole) [inline kelvinmpham.md:6]: Please replace the generic header with specific entries before merging.  Thanks!

**Review by nttoole — CHANGES_REQUESTED** [review]

**Review by i3ince — COMMENTED** [review]
Cool :dark_sunglasses:

**Comment by kelvinmpham** [comment]
@nttoole updated with more details

**Pull request merged**